components/04-views/product-view/product-view.hbs

```diff
@@ -0,0 +1,124 @@
+{{ render '@header' }}
+
+<div class="product-view container">
+    {{render '@breadcrumbs' }}
+
+    <div class="product-view__wrapper">
+        <div class="product-view__gallery">
+            {{ render '@gallery' }}
+        </div>
+
+        <div class="product-view__main-details">
+            <h1 class="heading heading--first-level product-view__title">
+                {{{ title }}}
+            </h1>
+
+            <span class="product-view__sku">
+                {{{ sku.label }}} {{{ sku.value }}}
+            </span>
+
+            <span class="product-view__price">
+                {{{ price }}}
+            </span>
+
+            <div class="
+                     product-view__stock-status
+                     {{#if stock.outOfStock}}
+                         product-view__stock-status--out-of-stock
+                     {{/if}}
+                     {{#if stock.alert}}
+                         product-view__stock-status--alert
+                     {{/if}}
+            ">
+                {{{ stock.html }}}
+            </div>
+
+            <div class="product-view__rating">
+                <div class="product-view__rating-rates">
+                    {{ render '@rating--summary' }}
+                    {{#if rating.parenthesis}}
+                        (
+                    {{/if}}
+                    <a href="{{ rating.href }}"
+                       class="product-view__link product-view__link--rating {{ rating.class }}"
+                    >
+                        {{{ rating.value }}}
+                    </a>
+                    {{#if rating.parenthesis}}
+                        )
+                    {{/if}}
+                </div>
+                <div class="product-view__rating-actions">
+                    <a href="{{ addReview.href }}"
+                       class="
+                           product-view__link
+                           product-view__link--add-review
+                           {{ addReview.class }}
+                       "
+                   >
+                        {{ addReview.text }}
+                    </a>
+                </div>
+            </div>
+        </div>
+
+
+        <div class="product-view__secondary-details">
+
+            <div class="product-view__description">
+                {{{ description }}}
+            </div>
+
+            <div class="product-view__options">
+                {{#each options as |option|}}
+                    {{> '@swatch' option}}
+                {{/each}}
+            </div>
+
+            <div class="product-view__main-actions">
+                <div class="product-view__qty">
+                    <h3>Qty</h3>
```

Comment:
IMO `h3` is not proper tag for qty

Reply:
which one is? I saw h5 in default swatches config so it would be for COlor / sizes. Here I have h2  color/sizes. What would you suggest span? @anqaka what about a11y point of view as I know some readers can jump trough headings to skip content. Do you know best practices for html structure related to a11y for product page?

Reply:
It depends on the place on the website - basically we shouldn't use `h3`, `h4`, `h5` for styling. Heading set a semantic value and we should keep its order... If qty it's after `h2` and its important we can keep `h3` tag. But the most important is order. If you don't know the order, set `div` or `span` and if you want this element focusable add `tabindex` property. In swatches I asked to make swatches heading to be set in config in case we will be using different ones.

Reply:
SO i guess it will be handle/changed if required during implementation per project - but I think later we should also check and improve our structure here as well